assignments/life_expectancy/cleaning.py

```diff
@@ -103,9 +105,9 @@ def main(input_path: str|Path,
     parser.epilog = "This is where the command-line utility's epilog goes."
     parser.add_argument('-i', default = f'{OUTPUT_DIR}/eu_life_expectancy_raw.tsv', help="You need to put here the path of the input file")
     parser.add_argument('-d', default= "[\t]", help = "Delimiter.")
-    parser.add_argument('-r', default= 'PT', help = "Filter for the region you want to select.")
-    parser.add_argument('-o', default = f'{OUTPUT_DIR}/pt_life_expectancy.csv', help="You need to put here the path where you want to write the output file")
+    parser.add_argument('-r', type = Region, choices = list(Region), default = Region.PT, help = "Filter for the region you want to select.")
```

Comment:
Passing Region to a list it is not necessary, and it works without it.
```suggestion
    parser.add_argument('-r', type = Region, choices = Region, default = Region.PT, help = "Filter for the region you want to select.")
```